tests/docker/ducktape-deps/tinygo-wasi-transforms

```diff
@@ -14,6 +14,10 @@ tinygo build -target wasi -opt=z \
   -panic print -scheduler none \
   -o "/opt/transforms/tinygo/identity_logging.wasm" ./identity_logging
 
+tinygo build -target wasi -opt=z \
+  -panic print -scheduler none \
+  -o "/opt/transforms/tinygo/tee.wasm" ./tee
```

Comment:
question: where's this located? I don't see it in the tree

Reply:
I honestly have a hard time navigating all this sometimes

https://github.com/redpanda-data/redpanda/blob/dev/src/transform-sdk/go/transform/internal/testdata/tee/transform.go

Reply:
Ah, duh. I make this mistake every time I go looking for one of these

Reply:
Open to suggestions for more intuitive locations

Reply:
It's not the location, it's the fact that I consistently `find src/transform-sdk -name <name>.go` instead of searching for the directory. Just user error; nbd.